packages/shared-internals/package.json

```diff
@@ -31,13 +31,15 @@
     "babel-import-util": "^2.0.0",
     "debug": "^4.3.2",
     "ember-rfc176-data": "^0.3.17",
-    "js-string-escape": "^1.0.1",
-    "resolve-package-path": "^4.0.1",
-    "typescript-memoize": "^1.0.1",
     "fs-extra": "^9.1.0",
+    "is-subdir": "^1.2.0",
+    "js-string-escape": "^1.0.1",
     "lodash": "^4.17.21",
     "minimatch": "^3.0.4",
-    "semver": "^7.3.5"
+    "pkg-entry-points": "^1.1.0",
```

Comment:
do we really need/want to add packages that are not completely vetted by us with little usage?
as this is a dependency that would end up in consumer build pipelines I believe we need to hold a high standard

Reply:
> do we really need/want to add packages that are not completely vetted by us with little usage?

It is already used by ember-auto-import! But fwiw it was me adding that in aforementioned PR (https://github.com/embroider-build/ember-auto-import/pull/623)... 🤷 😅

> as this is a dependency that would end up in consumer build pipelines I believe we need to hold a high standard

How would you define that standard then? 

It has a proper setup and tests (I know because I contributed the sync API that we use here). And the author is a reputable figure in OSS AFAIK, having created popular packages like `tsx`.